doc/param.md

```diff
@@ -0,0 +1,40 @@
+@function can-route.param param
+@parent can-route.static
+
+@description Get a route path from given data.
```

Comment:
Get a route path `from` given data.

`from`?

Reply:
To be more specific ... given the "state" you would like, creates a url fragment that represents that state.

I'm a bit wary of using `route` here ... `route` is something you pass to `.register()`.